joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitor.scala

```diff
@@ -1698,7 +1698,8 @@ class PythonAstVisitor(
           xMayHaveSideEffects = !attribute.value.isInstanceOf[ast.Name],
           lineAndColOf(call),
           argumentNodes,
-          keywordArgNodes
+          keywordArgNodes,
+          Some(relFileName + ":<inserted>.<module>.")
```

Comment:
Possibly don't add this kind of full name under `methodFullName` but under `dynamicTypeHints` instead. Having something like `<inserted>` here could result in some undesired behaviour down the line.